examples/python/migraphx_bert.py

```diff
@@ -0,0 +1,260 @@
+# Copyright 2022 Advanced Micro Devices, Inc.
```

Comment:
Why are there two sets of licenses in this file? If you've copied this from somewhere, keep the original license

Reply:
inserted MIT license (only) in all examples